Get current namespace when parent namespace is empty string

ibash opened the pull request:
This fixes issue: #533
Originally pull request #630

herom commented:
Thanks a lot @ibash, please merge your commits into a single one so I can merge your PR as requested in our [Guidelines for Submitting a Pull Request](https://github.com/vpulim/node-soap/blob/master/CONTRIBUTING.md#submitting-a-pull-request) :+1:

ibash commented:
Squashed and rebased onto latest master. @herom I noticed that my last build failed on node 12: https://travis-ci.org/vpulim/node-soap/jobs/63788061
I just tried it locally and it seemed to pass - if the current one fails can you help me debug?

Thanks!

herom commented:
Thanks @ibash :+1: 

I've seen TravisCI builds fail without any reason (last time it failed when testing a PR that only changed our license entry in the `package.json`) - this one seems to pass now :smiley_cat:
